src/nxt_process_title.c

```diff
@@ -130,6 +125,11 @@ nxt_process_arguments(nxt_task_t *task, char **orig_argv, char ***orig_envp)
         goto done;
     }
 
+    p = nxt_malloc(strings_size);
```

Comment:
If you move `p` after the other `goto done`, you'll arrive at `nxt_free()` with an uninitialized pointer, which will result in Undefined Behavior.